89.md

```diff
@@ -0,0 +1,62 @@
+NIP-89
+======
+
+Bitcoin Transaction Broadcasting Events
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains an encoded bitcoin transaction.
+The bitcoin transaction is encoded as a base64 string, and is contained in the `content` field.
+
+The network magic is contained in the `tags` field. This is to identify the network that the transaction is for.
+The common ones are:
+
+- mainnet: f9beb4d9
+- testnet3: 0b110907
+- signet (default): 0a03cf40
+- regtest: fabfb5da
+
+If another value is present it is likely a custom signet.
+
+This is an ephemeral event, and is not stored on the server. So relays or other custom clients could receive these
+events, and broadcast them to the bitcoin network.
+
+For example:
+
+```json
+{
+  "kind": 28333,
+  "content": "AgAAAAABAtkXo8t2LfqIwQ+BPQg/LZdqAtAb61bY0zIjFvvxMTtGAAAAAAD9////j8wvOaFL5Dz+0CTKXNKxxv1GrusMps3aodsXQGp95xgAAAAAAP3///8CLAPTBQAAAAAiUSBIKoy1/JK1GdfWhii8KLfBooZg3qT7ImuwwwC6t2eT6oiKAQAAAAAAIlEgZtFI4Bx2rDqBe4X9+lzEhh7Kxca3tNE9l/QfH/ZghQECRzBEAiBV2O5+jskV8/H+UuzgRuku2gCAwlmErieXrS5scbongAIgLWD6uN8BGpyE7gNtX4jxnkNShY8yjhdcjhsHR4RAWJEBIQLlCnhSR+K9YoAngUWj94mptHhcVYJGi+rT3lwApIekkQJHMEQCIDwrK0YHftZ2BQsBjHdJb89Tlpzvc3z8E1wGgbqIVRBQAiB6J0UXuPjib+geVmaZeoq4vYD+fDz835o5OJRg9ITFDAEhA1oq2Zj0d4XbATjc3OUSXSJDvvIosDeAsHyegwk4fkcTNAslAA==",
```

Comment:
Is it useful to make this an array so that the event can contain transaction packages right from the start?

Reply:
Very interesting idea! We could make the content a json array of transactions to support this. Another alternative could be to use tags, either using an `e` tag to reference another bitcoin transaction event or add a new tag for a parent transaction.

Reply:
It's better to use tags for everything so the content can be something stupid like the string `"a bunch of bitcoin transactions to be broadcasted"` that can be shown on nonsupporting clients.

Reply:
Not sure about the `e` tag to reference another bitcoin transaction event. It seems to complicate implementations again similar to bitcoin p2p, because they need to look for / wait for the referred transaction before making a decision whether the complete package is worth including in the mempool.